.github/workflows/tweet-issue.yml

```diff
@@ -0,0 +1,46 @@
+name: "Tweet issue/pr when labeled"
+
+on:
+  issues:
+    types: labeled
+  pull_request:
+    types: labeled 
+
+    
+permissions:
+  contents: read
+  issues: write
+  
+jobs:
+  tweet:
+    if: ${{ github.event.label.name == 'issue/tweet' }}
+    runs-on: ubuntu-latest
+    steps:
+        - name: Check if the workflow initiater is an organization member
+          uses: tspascoal/get-user-teams-membership@v2
+          id: checkUserMember
+          with:
+            username: ${{ github.actor }}
+            organization: "layer5"
+            team: "maintainers"
+            GITHUB_TOKEN: ${{ secrets.GLOBAL_TOKEN }}
+
+        - if: ${{ github.event_name != 'pull_request' && steps.checkUserMember.outputs.isTeamMember == 'true' }}
+          name: send a tweet for issue
+          uses: ethomson/send-tweet-action@v1
+          with:
+            status: "Hey everyone, we need some help with this issue - ${{ github.event.issue.title }} here at : ${{ github.event.issue.html_url }} #opensource"
```

Comment:
Update the wording here to be similar to the other PR we have under the meshery repo.